### PR TITLE
[Feature] Start/Join a Petition re-direct

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes #177392943

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/ActNowCard/PetitionCard.js [hunk 1]
```diff
@@ -18,11 +20,17 @@ function PetitionCard({ closeCard, promiseActNow, ...props }) {
 
   const { petitionJoin, petitionTitle: petitionStart } = props;
   const [open, setOpen] = useState(false);
+  const { session } = useSession();
```
Comment (kilemensi) [inline src/components/ActNowCard/PetitionCard.js:23]: I don't think the prop is called `session`.

File: src/components/ActNowCard/PetitionCard.js [hunk 2]
```diff
@@ -18,11 +20,17 @@ function PetitionCard({ closeCard, promiseActNow, ...props }) {
...
 
   const classes = useStyles();
 
   const handleFormOpen = () => {
-    setOpen(true);
+    if (!session) {
+      setOpen(false);
```
Comment (kilemensi) [inline src/components/ActNowCard/PetitionCard.js:29]: Form should be closed at this point, no?

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: src/components/ActNowCard/PetitionCard.js (hunk 1 above)
Comment (KobbyMmo) [reply src/components/ActNowCard/PetitionCard.js:23]: its `data` @nerdlet

File: src/components/ActNowCard/PetitionCard.js (hunk 2 above)
Comment (nerdlet) [reply src/components/ActNowCard/PetitionCard.js:29]: @kilemensi Correct 🚀

File: src/components/ActNowCard/PetitionCard.js (hunk 1 above)
Comment (nerdlet) [reply src/components/ActNowCard/PetitionCard.js:23]: Yap, @KobbyMmo @kilemensi  it's data

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**